Tools/Scripts/libraries/webkitcorepy/webkitcorepy/credentials.py

```
@@ -29,6 +29,15 @@
 _cache = dict()
 
 
+def handle_keyring_error(error):
+    if Terminal.choose(
+        f'Could not access credentials from keychain. Please run `security unlock-keychain` before continuing.',
+        default='Continue',
+        options='Continue',
+    ) == 'Continue':
+        pass
```

I think I might prefer failing here, instead of prompting for continuation? Especially since the remedy is to run another terminal command.

---

That makes a lot of sense, especially since this happens during ssh sessions where it's harder to just open a new terminal window!